source/_docs/drupal-s3.md

```diff
@@ -0,0 +1,222 @@
+9. Navigate to the group created in step 5 and select **Add Users to Group**.
+10. Select your newly created user and click **Add Users**.
+
+##Integrating S3 with Drupal
+You will need to install the appropriate Drupal module(s) and the AWS SDK library.
+
+#### Drupal 7
+The following instructions utilize [Terminus](/docs/terminus), Pantheon's CLI which allows you to call Drush remotely without using a local installation.
+
+These steps require Drush 8, which is run by default on Pantheon for newly created Drupal sites. Sites created prior to November 4, 2015 run 5.x by default.
+
+Before you begin:
+
+- [Modify the default Drush version](/docs/drush-versions#modifying-default-drush-version) to Drush 8 if needed.
+- Create a `sites/default/settings.php` file if you have not done so already.
+- Set the site's connection mode to SFTP within the site Dashboard or via [Terminus](/docs/terminus):
+
+ ```bash
+ terminus site set-connection-mode --mode=sftp
+ ```
+
+#### Drupal 8
+Before you begin:
+
+- Install [Composer](https://getcomposer.org/doc/00-intro.md#installation-linux-unix-osx) locally.
+- Set the site's connection mode to Git within the site Dashboard or via [Terminus](/docs/terminus):
+
+ ```bash
+ terminus site set-connection-mode --mode=git
+ ```
+
+###Install Required and Recommended Modules
+####S3 File System
+ <!-- Nav tabs -->
+ <ul class="nav nav-tabs" role="tablist">
+  <li role="presentation" class="active"><a href="#d7s3fs" aria-controls="d7s3fs" role="tab" data-toggle="tab">Drupal 7</a></li>
+  <li role="presentation"><a href="#d8s3fs" aria-controls="d8s3fs" role="tab" data-toggle="tab">Drupal 8</a></li>
+ </ul>
+ <!-- Tab panes -->
+ <div class="tab-content">
+  <div role="tabpanel" class="tab-pane active" id="d7s3fs">
+  <br/>
+  Install the <a href="https://www.drupal.org/project/libraries">Libraries API</a> and <a href="https://www.drupal.org/project/s3fs">S3 File System</a> modules:
+  <pre><code class="bash hljs">
+  terminus drush 'en libraries s3fs -y'
+  </code></pre>
+
+  Get the <a href="https://github.com/aws/aws-sdk-php/releases">AWS SDK Library 2.x</a>:
+  <pre><code class="bash hljs">
+  terminus drush 'make --no-core sites/all/modules/contrib/s3fs/s3fs.make'
+  </code></pre>
+  The above command will add the AWS SDK version 2.x into the <code>sites/all/modules/contrib/s3fs</code> directory.
+ </div>
+  <div role="tabpanel" class="tab-pane" id="d8s3fs">
+   <br/>
+   <p>Install the <a href="https://www.drupal.org/project/s3fs">S3 File System</a> module and AWS SDK version 3.x library using Composer.</p>   
+   First, ensure that the Composer for your site will first look to Drupal's preferred package source to find modules:
+   <pre><code class="bash hljs">
+   composer config repositories.drupal composer https://packages.drupal.org/8
+   </code></pre>
+   Install s3fs module from the preferred package source:
+   <pre><code class="bash hljs">
+   composer require drupal/s3fs --prefer-dist
```

Comment:
@rachelwhitton In D8 the standard installation of D8 modules and their required libraries via Composer should need no additional special configuration. How to handle the vendor directory is actually up to the user and their workflow pattern, see: [Decide Whether to Commit the Vendor Directory|https://pantheon.io/docs/composer-drupal-8/#decide-whether-to-commit-the-vendor-directory].